Rakefile

```diff
@@ -538,7 +538,28 @@ end
 
 installed_pod_bin = File.join(BUNDLE_DESTROOT, 'bin/pod')
 file installed_pod_bin => rubygems_update_dir do
-  install_gem 'cocoapods', install_cocoapods_version
+  execute "bundling", ["rm", "-rf", WORKBENCH_DIR + "/ruby"]
+  # Can figure this one out later
+  bundler = "/Users/orta/.rvm/gems/ruby-2.1.3/wrappers/bundle"
```

Comment:
Perhaps via $GEM_HOME?

```
$ echo $GEM_HOME
/Users/kyle/gems
$ ls $GEM_HOME
bin            bundler        doc            specifications
build_info     cache          gems
```

Remember to fall back to correct path when GEM_HOME isn't set. There is probably some RubyGems/Bundler code to look up this place.